src/sentry/deletions/defaults/group.py

```diff
@@ -131,48 +104,9 @@ def tenant_ids(self) -> Mapping[str, Any]:
     def chunk(self) -> bool:
         """This method is called to delete chunks of data. It returns a boolean to say
         if the deletion has completed and if it needs to be called again."""
-        if not options.get("deletions.nodestore.parallelization-task-enabled"):
-            events = self.get_unfetched_events()
-            if events:
-                # Adding this variable to see the values in stack traces
-                last_event = events[-1]
-                self.delete_events_from_nodestore_sequential(events)
-                # This value will be used in the next call to chunk
-                self.last_event = last_event
-                # As long as it returns True the task will keep iterating
-                return True
-            else:
-                # Now that all events have been deleted from the eventstore, we can delete the events from snuba
-                self.delete_events_from_snuba()
-                return False
-        else:
-            self.delete_events_from_nodestore_and_eventstore()
-            return False
-
-    # XXX: To be removed
-    def delete_events_from_nodestore_sequential(self, events: Sequence[Event]) -> None:
-        # We delete by the occurrence_id instead of the event_id
-        node_ids = [
-            Event.generate_node_id(
-                event.project_id,
-                (
-                    event._snuba_data["occurrence_id"]
-                    if self.dataset == Dataset.IssuePlatform
-                    else event.event_id
-                ),
-            )
-            for event in events
-        ]
-        nodestore.backend.delete_multi(node_ids)
-        self.post_delete_events_from_nodestore(events)
-
-    # XXX: To be removed
-    def post_delete_events_from_nodestore(self, events: Sequence[Event]) -> None:
-        pass
-
-    # XXX: To be removed
-    def delete_events_from_snuba(self) -> None:
-        raise NotImplementedError
+        self.delete_events_from_nodestore_and_eventstore()
+        self.delete_events_from_snuba()
```

Comment:
In the previous PR I lost this line.
I probably messed something up when writing the [test_issue_platform_batching_with_new_task](https://github.com/getsentry/sentry/pull/96795/files#diff-f4189429fef62000ea22c442fa8e52c3ba2877d16fc3fe9b88254b9215660012R431) test.

No customer impact from this mistake. A lot of events are probably lingering in Snuba that will be deleted with the 90 days TTL.